requirements/dev.txt

```diff
@@ -1,3 +1,4 @@
+glpk>=0.4.6
```

Comment:
what is this?
remove this from dev, if it is needed it should only be in bench.txt